config/routes.rb

```diff
@@ -14,37 +14,81 @@
     as: :ontology_iri_versioned,
     constraints: [
       RefLocIdRouterConstraint.new(Ontology, ontology: :id),
-      MIMERouterConstraint.new('text/plain', 'text/html')
-    ]
+    ] do
+      accept 'text/html'
+      accept 'text/plain'
+      accept 'application/json'
+
+      doc title: 'Ontology IRI (loc/id) with version reference',
+          body: <<-BODY
+Will return a representation of the ontology at a
+ontology version referenced by the {reference}.
+      BODY
+    end
 
   # MMT-Support
   specified_get '/ref/mmt/:repository_id/*path' => 'ontologies#show',
     as: :ontology_iri_mmt,
     constraints: [
       MMTRouterConstraint.new(Ontology, ontology: :id),
-      MIMERouterConstraint.new('text/plain', 'text/html')
-    ]
+    ] do
+      accept 'text/html'
+      accept 'text/plain'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.